include/amici/model.h

```diff
@@ -599,7 +612,21 @@ class Model : public AbstractModel {
     virtual std::vector<std::string> getObservableIds() const;
 
     /**
-     * @brief Checks whether the defined noise model is gaussian, i.e., the nllh is quadratic
+     * @brief Report whether the model has expression IDs set.
+     * @return Boolean indicating whether expression ids were set. Also returns
+     * `true` if the number of corresponding variables is just zero.
+     */
+    virtual bool hasExpressionIds() const;
+
+    /**
+     * @brief Get IDs of the expression.
+     * @return Expression IDs
+     */
+    virtual std::vector<std::string> getExpressionIds() const;
```

Comment:
We could consider changing that at some point to `getIdsFor({observables,states,expressions})` etc. That would save quite a bit of more or less replicated code...